mining/mining.go

```diff
@@ -206,10 +207,12 @@ func (g *BlkTmplGenerator) NewBlockTemplate(payToAddress util.Address, extraNonc
 		return nil, err
 	}
 
+	targetDifficulty := big.NewInt(0)
+	util.CompactToBig(msgBlock.Header.Bits, targetDifficulty)
 	log.Debugf("Created new block template (%d transactions, %d in fees, "+
 		"%d mass, target difficulty %064x)",
 		len(msgBlock.Transactions), txsForBlockTemplate.totalFees,
-		txsForBlockTemplate.totalMass, util.CompactToBig(msgBlock.Header.Bits))
+		txsForBlockTemplate.totalMass, targetDifficulty)
```

Comment:
It might be a good idea to use LogClosure here, so that targetDifficulty will not be calculated if log level is above debug